spec/requests/api/v2/access_request_spec.rb

```diff
@@ -377,4 +391,21 @@
       end
     end
   end
+
+  describe "DELETE #destroy" do
+    let(:first_access_request) do
+      access_request = create(:access_request)
+      access_request.discard
+      access_request
+    end
+
+    before do
+      delete "/api/v2/access_requests/#{first_access_request.id}",
+             headers: { "HTTP_AUTHORIZATION" => credentials }
+    end
+
+    it "should add a discarded_at timestamp" do
+      expect(first_access_request.discarded_at).to be_within(1.second).of(Time.now.utc)
```

Comment:
I feel like this is where @danielburnley's comment applies ... it'd be better to test that `access_request` doesn't show up in `AccessRequest.all` or `User.access_requests`, whichever we're more likely to use to get access requests.

Reply:
I have a test for this already.